src/hooks/useGetLocation.ts

```diff
@@ -0,0 +1,26 @@
+import { useGeolocated } from 'react-geolocated';
+
+type GeolocationCoordinates = {
+  latitude: number;
+  longitude: number;
+  accuracy: number;
+  altitude: number | null;
+  altitudeAccuracy: number | null;
+  heading: number | null;
+};
+
+const useGetLocation = (): {
+  coords: GeolocationCoordinates | undefined;
+  isGeolocationAvailable: boolean;
+  isGeolocationEnabled: boolean;
+} => {
+  const { coords, isGeolocationAvailable, isGeolocationEnabled } = useGeolocated({
```

Comment:
no need to use separate external package for getting user coordinates. please refactor the code and remove 'react-geolocated' package